.azure-pipelines/scripts/test_runner.sh

```diff
@@ -183,7 +200,7 @@ suite_test_start_time=$(date +%s)
 
 failure_identifiers=()
 while IFS= read -r line; do failure_identifiers+=("$line"); done < "$failure_identifiers_file"
-
+echo "This is Nightly Build: $SGXLKL_NIGHTLY_BUILD"
```

Comment:
Confusing message. Suggest rewording.

Reply:
Done removed. This message is not necessary actually.